### ISSUE TITLE
Update alt text for Footer icons

### ISSUE THREAD

**Issue by tonynguyen111997** [inland-empire-software-development/landing#26]
Alt text for social network icon images in Footer component are missing.

**Comment by hylickipiotr** [comment]
I would like to add alt text to footer icons 👋

**Comment by zootechdrum** [comment]
hey Tony can I would like to volunteer to take this on. Is this still available?

**Comment by tonynguyen111997** [comment]
Hey Cesar,

Sorry, but it looks like Piotr (nexus0212) asked first.
